### PR TITLE
Make Responsive Footer #2

### LINKED ISSUE #2: [feature]: Make responsive footer

### PR DESCRIPTION
**Pull request by kartikayasija**
closes #2 

Tech Used-
`HTML` `CSS`

Additional Library- 
`Font Awesome`

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,10 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
```
Comment (AbhiPatel10) [inline index.html:9]: Do not use Bootstrap; make it in HTML, CSS, and JavaScript.

Use only the Tech Stack, which is written in the readme file.

**Review by AbhiPatel10 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kartikayasija) [reply index.html]: I have updated the code, check it out.

**Comment by AbhiPatel10** [comment]
Put Company name and Icons on left side

**Comment by kartikayasija** [comment]
Updated Footer-
-> Added color to social links hover
-> Added Copyright Section
-> Updated the font family and made the color a little darker


image



image

**Comment by kartikayasija** [comment]
> Put Company name and Icons on left side

Okay

**Comment by kartikayasija** [comment]
Updated Footer-

- Added colors to social links hover
- Added Copyright section
- Updated company name and icon to left side
- Updated Font family and made the font darker


image



image

File: css/style.css [hunk 2]
```diff
@@ -1,9 +1,164 @@
...
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+ul {
+  list-style: none;
+}
```
Comment (AbhiPatel10) [inline css/style.css:30]:
```suggestion

a{
  text-decoration: none;
}
ul {
  list-style: none;
}

```

Put ui under a and below ui Comment that footer CSS starting from hear. and end with here.

Do this after that good to go

Comment (kartikayasija) [reply css/style.css]: Done

**Pull request merged**